time-it-macros/src/lib.rs

```diff
@@ -75,14 +94,15 @@ pub fn time_fn(_: TokenStream, item: TokenStream) -> TokenStream {
 
     let start_ident = get_random_name();
 
-    let msg = old_name.to_string(); //TODO: allow for optional msg arg
+    let tag = old_name.to_string(); //TODO: allow for optional msg arg
```

Comment:
is this TODO still relevant?

Reply:
yes this is for a custom message in `#[time_fn("msg")]`